mailpoet/views/emails/statsNotification.html

```
@@ -374,6 +374,21 @@ <h2 style="text-align:center;padding:0;font-style:normal;font-weight:normal;marg
       </table>
     </td>
   </tr>
+  <% if notTracked > 0 %>
+    <tr>
+      <td class="mailpoet_content" align="center" style="border-collapse:collapse">
+        <table width="100%" border="0" cellpadding="0" cellspacing="0" style="border-spacing:0;mso-table-lspace:0;mso-table-rspace:0;border-collapse:collapse">
+          <tbody>
+          <tr>
+            <td class="mailpoet_paragraph mailpoet_padded_side" style="word-break:break-word;word-wrap:break-word;text-align:center;border-collapse:collapse;color:#7f7f7f;font-family:Arial,'Helvetica Neue',Helvetica,sans-serif;font-size:13px;line-height:20px;padding-left:20px;padding-right:20px;padding-bottom:10px">
+              <%= __('%1$s of your recipients are not tracked, so open and click rates are based on the other %2$s.')|replace({'%1$s': notTracked, '%2$s': trackedSent}) %>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)